Services/Notifications/classes/class.ilObjNotificationAdminAccess.php

```diff
@@ -1,32 +1,11 @@
 <?php
```

Comment:
Please add `strict_types` (for every file in `Services/Notifications`).

Reply:
Same here. I would outsource this into a major refactoring/PHP8 PR